closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -855,6 +857,18 @@ JNIEXPORT void JNICALL JNI_OnUnload(JavaVM * vm, void * reserved)
     crypto_library = NULL;
 }
 
+/* Check whether MD5 is available.
+ *
+ * Class:     jdk_crypto_jniprovider_NativeCrypto
+ * Method:    isMD5Available
+ * Signature: ()Z
+ */
+JNIEXPORT jboolean JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_isMD5Available
+  (JNIEnv *env, jclass thisObj)
+{
+    return (NULL != OSSL_md5);
```

Comment:
The name `thisObj` is inappropriate (this related to a `static` method); suggest 'clazz` or `thisClass`.

The return expression should explicitly use constants appropriate for `jboolean`:
```c
    return (NULL != OSSL_md5) ? JNI_TRUE : JNI_FALSE;
```

Reply:
Changed.